internal/pkg/deploy/cloudformation/cloudformation_integration_test.go

```diff
@@ -258,6 +259,93 @@ func Test_Project_Infrastructure(t *testing.T) {
 			validationFunction(output)
 		}
 	})
+
+	t.Run("Deploys supporting infrastructure for pipeline (KMS Key, S3 Bucket)", func(t *testing.T) {
+		project := archer.Project{Name: randStringBytes(10), AccountID: callerInfo.Account}
+		projectRoleStackName := fmt.Sprintf("%s-infrastructure-roles", project.Name)
+		projectStackSetName := fmt.Sprintf("%s-infrastructure", project.Name)
+
+		// Make sure we delete the stacks after the test is done
+		defer func() {
+			// Clean up any StackInstances we may have created.
+			if stackInstances, err := cfClient.ListStackInstances(&awsCF.ListStackInstancesInput{
+				StackSetName: aws.String(projectStackSetName),
+			}); err == nil && stackInstances.Summaries != nil && stackInstances.Summaries[0] != nil {
+				projectStackInstance := stackInstances.Summaries[0]
```

Comment:
There will only ever be 1 StackInstance Summary?

Reply:
yea we will only create one stack instance in one region per project, regardless of how many pipeline you create in that region. On ln 323 we assert that.